### ISSUE TITLE
Failing to install on Windows either with yarn or npm

### ISSUE THREAD

**Issue by tinovyatkin** [jkimbo/eslint-plugin-jest#2]
Hello,

Not sure what is the idea behind checking for `yarn` at `postinstall` however, it fails to install either with `npm` or `yarn` on Windows.

`yarn` error:

```
PS C:\Users\konst\Sites\postcss-amp> yarn install
yarn install v0.17.8
info No lockfile found.
[1/4] Resolving packages...
warning jest > jest-cli > istanbul-api > fileset > minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[2/4] Fetching packages...
warning fsevents@1.0.15: The platform "win32" is incompatible with this module.
info "fsevents@1.0.15" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
⠁
⠁
⠁
⠁
error C:\Users\konst\Sites\postcss-amp\node_modules\eslint-plugin-jest: Command failed.
Exit code: 1
Command: C:\WINDOWS\system32\cmd.exe
Arguments: /d /s /c if [[ $npm_execpath != *'yarn.js'* ]]; then echo 'ERROR: Use yarn to install dependencies: `npm install -g yarn && yarn` (https://yarnpkg.com)' && exit
Directory: C:\Users\konst\Sites\postcss-amp\node_modules\eslint-plugin-jest
Output:
$npm_execpath was unexpected at this time.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by jkimbo** [comment]
@tinovyatkin sorry about that, it was an experiment to ensure that anyone developing on the project would use yarn, however it was rather an ill fated one! I'll remove it.

**Comment by jkimbo** [comment]
Published as v1.0.2
